### PR TITLE
Upgrade to SpatialOS 13.0.0

### PR DESCRIPTION
**Pull request by improbable-valy**
Upgrade to SpatialOS 13.0.0 and migrate to the Unreal SDK 1.0.0 using [this guide](https://github.com/spatialos/UnrealSDK/blob/master/docs/migration.md).
Tests performed:
* Building and running the project locally
* Deploying to the cloud and starting a client from the launcher
  * Known issue: the client can't connect to a cloud deployment when started from the launcher. Workaround: apply the patch from [this PR](https://github.com/EpicGames/UnrealEngine/pull/4206) in your UnrealEngine checkout

### CODE REVIEW

**Review by jared-improbable — CHANGES_REQUESTED** [review]
The Generated/ and Plugins/*/Binaries/ folders shouldn't be committed.

File: workers/unreal/Game/Plugins/SpatialOS/SpatialOSEditorToolbar/Binaries/Win64/UE4Editor.modules [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (jared-improbable) [inline workers/unreal/Game/Plugins/SpatialOS/SpatialOSEditorToolbar/Binaries/Win64/UE4Editor.modules]: This folder needs ignoring, too

**Review by jared-improbable — APPROVED** [review]

**Review by jared-improbable — APPROVED** [review]

**Review by ElleEss — APPROVED** [review]
I checked the link, just in case. (Obsessed with broken links.)

**Review by ElleEss — APPROVED** [review]
Be good to use the 'SpatialOS Unreal SDK' name that's in the docs to avoid confusion. See comment in-line.

**Review by oblm — APPROVED** [review]

**Pull request merged**